app/views/directives/osc-routing.html

```diff
@@ -155,6 +155,47 @@
       </div>
     </div>
 
+    <!-- If there is exactly one alternate service, let the user change the weights using a slider. -->
+    <div ng-if="route.alternateServices.length === 1 && !controls.hideSlider">
+      <h3>Service Weights</h3>
+      <div class="form-group">
+        <div class="weight-slider-values">
+          <div>
+            <span class="service-name">{{route.to.service.metadata.name}}</span>
+            <span class="weight-percentage">{{weightAsPercentage(route.to.weight)}}</span>
+          </div>
+          <div>
+            <span class="weight-percentage hidden-xs">{{weightAsPercentage(route.alternateServices[0].weight)}}</span>
+            <span class="service-name">{{route.alternateServices[0].service.metadata.name}}</span>
+            <span class="weight-percentage visible-xs-inline">{{weightAsPercentage(route.alternateServices[0].weight)}}</span>
+          </div>
+        </div>
+        <label class="sr-only" for="weight-slider">Service {{route.to.service.metadata.name}} Weight</label>
+        <input
+            id="weight-slider"
+            type="range"
+            min="0"
+            max="100"
+            step="1"
+            list="ticks"
+            ng-model="controls.rangeSlider"
+            aria-describedby="weight-slider-help"
+            class="mar-top-md">
+        <!-- Use class="hidden" to make sure the options don't render for browsers that don't support datalist. -->
```

Comment:
how does it look on the ones that do support, it looked like it was just Safari and iOS that didnt